assemblies/pentaho-war/pom.xml

```diff
@@ -50,6 +50,13 @@
     </dependency>
     <!-- endregion -->
 
+    <!-- Needed for runtime -->
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-context-support</artifactId>
```

Comment:
If this artifact is needed at runtime then scope should be runtime `<scope>runtime</scope>`.
This artifact is for runtime correct? Not tests?

Reply:
Yeap, just confirmed. I'll change it